src/decision/dMCDA.jl

```diff
@@ -751,27 +806,45 @@ function site_selection(domain::Domain, scenario::DataFrameRow, w_scens::NamedDi
     out_conn = out_conn[site_ids]
     strong_pred = strong_pred[site_ids]
 
-    (_, _, ranks) = guided_site_selection(mcda_vars, scenario.guided, true, true, prefseedsites, prefshadesites, rankingsin, in_conn, out_conn, strong_pred)
+    (_, _, ranks) = guided_site_selection(
+        mcda_vars,
+        scenario.guided,
+        true,
+        true,
```

Comment:
Avoid magic values - assign these to variables that at least indicate why they are true (or false)